GeometryCore/src/Polygon.cpp

```diff
@@ -252,4 +252,83 @@ namespace geom_utils
         return poly;
     }
 
+    inline FPoint2D vecRot90CW(const FPoint2D& point) 
+    {
+        FPoint2D newPoint(point.y,-point.x);
+        return newPoint;
+    }
+
+    inline FPoint2D vecRot90CCW(const FPoint2D& point) 
+    {
+        FPoint2D newPoint(-point.y,point.x);
+        return newPoint;
+    }
+
+    inline FPoint2D vecUnit(FPoint2D& point,const float value) 
```

Comment:
I suggest to create separate `FPoint2D normalize(FPoint2D vec)` function and move the first part of this function into it (put in linear algebra instead of Polygon.cpp).
The last part `FPoint2D d = (point * value);` is simple operation of multiplication. Not necessary to create separate function for this.

Learn to find and recognize common or general code, which can be used somewhere else and move these code parts into separate functions available for the other parts of program.